djconnectwise/sync.py

```diff
@@ -2228,7 +2203,7 @@ def sync_related(self, instance, sync_config={}):
 
         if sync_config.get('sync_service_note', True):
             note_sync = ServiceNoteSynchronizer()
-            note_sync.api_conditions = [instance_id]
+            note_sync.sync_single_id = instance_id
```

Comment:
At line 2216, do you think we can follow the same pattern of `note_sync.sync_single_id = instance_id`?

```
        if sync_config.get('sync_activity', True):
            activity_sync = ActivitySynchronizer()
            activity_sync.api_conditions = ['ticket/id={}'.format(instance_id)]
            sync_classes.append((activity_sync, Q(ticket=instance)))
```

Reply:
I thought about that as well. But no I don't think we would want to change `activity_sync` and `time_sync` to use `sync_single_id` because they actually use the instance_id in conditions they apply to the request. 

In the case of the service note related sync, we were adding the instance_id to the list of the conditions when it is not needed in the conditions at all - we had to remove it from the conditions and reset them before making the call. It was pretty messy.

Reply:
Okay, got it. Thanks.